model/authorizations/v1/access_review_request_type.model

```diff
@@ -25,9 +25,12 @@ struct AccessReviewRequest {
 	// Indicates which Organization the resource type belongs to
 	OrganizationID string
 
-	// Indicates which Cluster the resource type belongs to
+	// Indicates which Cluster (internal id) the resource type belongs to
 	ClusterID string
 
+	//Indicates which Cluster (external id) the resource type belongs to
```

Comment:
Add a space between `//` and `Indicates`. It makes style of comments more consistent with the rest of the code of the project. Same in the other files.